packages/lib/src/components/paywall/Subscriptions.svelte

```diff
@@ -101,9 +108,15 @@
             {/if}
           </Column>
 
-          <Button href={url} class="w-full mt-2" variant={tag ? 'primary' : 'secondary'}>
-            {buttonText || t('continue')}
-          </Button>
+          {#await getCheckoutUrl(subscription) then checkoutUrl}
+            <Button
+              href={checkoutUrl.replace('poId', 'option')}
```

Comment:
right, this is temporary until this one is merged: https://github.com/sesamyab/sesamy-js/pull/456